### ISSUE TITLE
「∥」symbol?

### ISSUE THREAD

**Issue by kcwu** [audreyt/moedict-webkit#104]
https://www.moedict.tw/#~腳
「指演員。∥現在一般寫作「角」。」
is 「∥」 intended symbol?

![screen shot 2013-12-21 at 10 12 58 am](https://f.cloud.github.com/assets/175566/1795037/7a1376ae-69e5-11e3-8edf-ebfe7c4d8d52.png)

**Comment by audreyt** [comment]
Yes. ∥ is intentional. In the cross-strait dictionary data it marks that whatever follows applies to all the senses listed above it; in this entry, 「現在一般寫作「角」」 qualifies the acting-related senses of 腳. As of 2026-07-10 the data still carries it (https://www.moedict.tw/api/c/腳.json), and the current frontend handles the layout explicitly: the text after ∥ is split out and rendered as its own block below the definition list, and on https://www.moedict.tw/~腳 that block renders inside the entry card. Since the symbol is by design and the layout question has been addressed in the current implementation, closing as completed.
